services/route_manager/src/main/java/com/futurewei/alcor/route/controller/RouteController.java

```diff
@@ -0,0 +1,78 @@
+package com.futurewei.alcor.route.controller;
```

Comment:
we will need the copy right header for each source file.

Reply:
Updated.